ddtrace/bootstrap/sitecustomize.py

```diff
@@ -142,6 +51,13 @@ def cleanup_loaded_modules(aggressive=False):
 
 log = get_logger(__name__)
 
+if os.environ.get("DD_GEVENT_PATCH_ALL", "false").lower() in ("true", "1"):
```

Comment:
👍  I think we can just check that `DD_GEVENT_PATCH_ALL` is in `os.environ` as we only care about whether it is set (to whatever) or not. Should we also include a test to check that we get the warning?